ReactWindows/ReactNative.Shared/CoreModulesPackageWrapper.cs

```diff
@@ -0,0 +1,424 @@
+                                Module.measureLayout(
+                                    args[0].ToObject<System.Int32>(),
+                                    args[1].ToObject<System.Int32>(),
+                                    new Callback(args[2].ToObject<int>(), instance),
+                                    new Callback(args[3].ToObject<int>(), instance)
+                                )
+                            )
+                        },
+                        {
+                            nameof(ReactNative.UIManager.UIManagerModule.measureLayoutRelativeToParent),
+                            new NativeMethod("async", (instance, args) =>
+                                Module.measureLayoutRelativeToParent(
+                                    args[0].ToObject<System.Int32>(),
+                                    new Callback(args[1].ToObject<int>(), instance),
+                                    new Callback(args[2].ToObject<int>(), instance)
+                                )
+                            )
+                        },
+                        {
+                            nameof(ReactNative.UIManager.UIManagerModule.findSubviewIn),
+                            new NativeMethod("async", (instance, args) =>
+                                Module.findSubviewIn(
+                                    args[0].ToObject<System.Int32>(),
+                                    (Newtonsoft.Json.Linq.JArray)args[1],
+                                    new Callback(args[2].ToObject<int>(), instance)
+                                )
+                            )
+                        },
+                        {
+                            nameof(ReactNative.UIManager.UIManagerModule.setJSResponder),
+                            new NativeMethod("async", (instance, args) =>
+                                Module.setJSResponder(
+                                    args[0].ToObject<System.Int32>(),
+                                    args[1].ToObject<System.Boolean>()
+                                )
+                            )
+                        },
+                        {
+                            nameof(ReactNative.UIManager.UIManagerModule.clearJSResponder),
+                            new NativeMethod("async", (instance, args) =>
+                                Module.clearJSResponder(
+                                )
+                            )
+                        },
+                        {
+                            nameof(ReactNative.UIManager.UIManagerModule.dispatchViewManagerCommand),
+                            new NativeMethod("async", (instance, args) =>
+                                Module.dispatchViewManagerCommand(
+                                    args[0].ToObject<System.Int32>(),
+                                    args[1].ToObject<System.Int32>(),
+                                    (Newtonsoft.Json.Linq.JArray)args[2]
```

Comment:
`args[2]` can be null

Reply:
Good point. Forgot about that.

Reply:
Pushed the commits again, added a static helper to `NativeModuleWrapperBase` called `CastJToken` which does a null or undefined check and returns null for the `JToken` in those cases. I actually should modify a bit and skip that for `JValue`...